components/Competition/CreateCompetition/index.js

```diff
@@ -26,42 +36,58 @@ export default function CreateCompetitionForm() {
     setCompetitionGameInput(Number(event.target.value));
   }
 
-  function handleSubmit(event) {
-    event.preventDefault();
```

Comment:
Why did you delete `event.preventDefault()`? Your app will be reloaded when you don't suppress the default behavior.

Reply:
In order to use the `StyledButtonComponent` (which comes with the toast message) on forms too I had to pass the button the submitHandleFunction in order to execute it after the timeout.
The form itself is still preventing the default like this `<form onSubmit={(event) => event.preventDefault()}`

It seems to work as intended but is this bad pratice?